src/FrameManager.ts

```diff
@@ -143,10 +146,13 @@ class FrameManager {
   public sendToClient<T, V>(message: PartialMsg<T, V>) {
     const clientOrigin = this._expectedClientOrigin();
     if (this._iframe.contentWindow && clientOrigin) {
-      let validated = null;
+      let validated: HostToClient;
 
       try {
         validated = validateOutgoing(message);
+        if (validated.direction === undefined) {
+          validated.direction = 'HostToClient';
```

Comment:
I'm fairly certain we don't need/want the conditional here. Every outgoing message from the host should be tagged as `HostToClient`. Only setting if it's undefined implies that the app interfacing with this might somehow be producing a message that has `direction` already set, which is probably not a good idea.